src/test/java/uk/gov/hmcts/reform/pip/data/management/service/artefactsummary/nonstrategic/RptEasternWeeklyHearingListSummaryDataTest.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.reform.pip.data.management.service.artefactsummary.nonstrategic;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.IOUtils;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.pip.data.management.service.ListConversionFactory;
+import uk.gov.hmcts.reform.pip.data.management.service.artefactsummary.ArtefactSummaryData;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.pip.model.publication.ListType.RPT_EASTERN_WEEKLY_HEARING_LIST;
+
+@ActiveProfiles("test")
+class RptEasternWeeklyHearingListSummaryDataTest {
```

Comment:
This 5 RPT tests are using the same test files. The only thing different is the list type. I think it might be better to have a single test class and use ParameterizedTest with EnumSource with loop through each list type in the test

Reply:
done.